app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -11,40 +11,57 @@ REVISION="1"
 SOURCE_URI="https://github.com/davidgiven/wordgrinder/archive/$portVersion.tar.gz"
 CHECKSUM_SHA256="54085af7963e1f67342bc0b1b20d1ccc75494f2e23d401b601cf3089acea747c"
 
-ARCHITECTURES="!x86_gcc2 !x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
 
 PROVIDES="
-	wordgrinder = $portVersion
-	cmd:WordGrinder = $portVersion
+	wordgrinder$secondaryArchSuffix = $portVersion
+	cmd:wordgrinder$commandSuffix = $portVersion
 	"
 REQUIRES="
-	haiku
-	lib:libncurses
-	lib:liblua
-	lib:libz
+	haiku$secondaryArchSuffix
+#	lib:libfreetype$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix >= 5.3
+	lib:libncurses$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
-	haiku_devel
-	devel:libncurses
-	devel:liblua
-	devel:libz
+	haiku${secondaryArchSuffix}_devel
+#	devel:libfreetype$secondaryArchSuffix
+	devel:liblua$secondaryArchSuffix >= 5.3
+	devel:libncurses$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:g++
+	cmd:cc$secondaryArchSuffix
 	cmd:make
 	cmd:ninja
-	cmd:pkg_config
+	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage wordgrinder$secondaryArchSuffix \
+	$commandBinDir/wordgrinder
+
 BUILD()
 {
-	make OBJDIR=$sourceDir/build PREFIX=$prefix BINDIR=$binDir \
-		DOCDIR=$docDir MANDIR=$manDir
+	make PREFIX=$prefix BINDIR=$commandBinDir DOCDIR=$docDir \
+		MANDIR=$manDir HOME=$`finddir B_USER_DIRECTORY` \
```

Comment:
Hmm, that's a good idea to put the call here, but there's a small error.
```diff
-		MANDIR=$manDir HOME=$`finddir B_USER_DIRECTORY` \
+		MANDIR=$manDir HOME="`finddir B_USER_DIRECTORY`" \
```
(same thing in INSTALL, of course.)

Reply:
Ah, k, will change, didn't produce an error here :)